### ISSUE TITLE
Please add POT files for support community translations

### ISSUE THREAD

**Issue by AlexGluck** [mchugh19/zabbix-vue#16]
I ready for translation to Russian.

**Comment by mchugh19** [comment]
That's excellent!

The chrome/firefox extension framework already includes internationalization. The default language for this project is in the en folder of [_locales](/mchugh19/zabbix-vue/tree/master/app/_locales). If you translate the message and description values in messages.json, they can be submitted as a PR under a new ru folder, or just attach to this issue, and I'd be happy to add it. 

Thanks much!

**Comment by mchugh19** [comment]
@AlexGluck are you able to see the messages.json file? All that has to be done is to translate those values. If you send send over the translated file, I can make a new release to support Russian.

**Comment by AlexGluck** [comment]
![2018 06 12-17 30 23-286x34](https://user-images.githubusercontent.com/8583390/41296782-5c359858-6e66-11e8-877d-d3b3014d952e.png)
Translation not placed to fields.

**Comment by mchugh19** [comment]
Is that a screenshot of the plugin? What do you mean the translation is not placed to fields? 

If your browser is set to use ru localization, then it should be pulling messages from the ru messages.json. Is this not working and falling back to english?

**Comment by AlexGluck** [comment]
yep, this is screenshot. String may be cuted in other translations(such as russian). Sorry for my bad english.

Here screenshot how looks settings window for your extension. Some fields not contain full text string.
![2018 06 12-19 51 05-380x602](https://user-images.githubusercontent.com/8583390/41304862-fca7a3ae-6e79-11e8-90a6-83d015a23997.png)

**Comment by mchugh19** [comment]
Long lines should now wordwrap in the options screen.
